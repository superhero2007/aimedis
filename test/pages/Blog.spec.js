import React from 'react';
import { shallow } from 'enzyme';

import Blog from 'pages/Blog';

function setup() {
  return shallow(<Blog />);
}

describe('Blog', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
