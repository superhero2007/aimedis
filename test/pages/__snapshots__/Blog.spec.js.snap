// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog should render properly 1`] = `"<div class=\\"app__blog app__route\\"><div class=\\"container\\"><section class=\\"app__blog__section1\\"><div class=\\"app__blog__section1__mobile d-block d-lg-none\\"><p class=\\"app__blog__section1__mobile__subtitle\\">Featured post</p><p class=\\"app__blog__section1__mobile__title\\">AR and VR are the new tools in your doctor’s bag</p></div><div class=\\"app__blog__section1__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-8 app__blog__section1__content__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"col-12 col-lg-4 app__blog__section1__content__info\\"><p class=\\"app__blog__section1__content__info__subtitle d-none d-lg-block\\">Featured post</p><p class=\\"app__blog__section1__content__info__title d-none d-lg-block\\">AR and VR are the new tools in your doctor’s bag</p><p class=\\"app__blog__section1__content__info__description\\">Medicine has always been an obscure discipline to practise. A doctor has been looking at with awe and respect.</p></div></div></div></section><section class=\\"app__blog__section2\\"><div class=\\"app__blog__list\\"><div class=\\"app__blog__list__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/1\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/2\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/3\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/4\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/5\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/6\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/7\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/8\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/9\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div></div></div><div class=\\"app__blog__list__pagination d-none d-lg-flex\\"><div class=\\"app__blog__list__pagination__item prev\\">&lt;</div><div class=\\"app__blog__list__pagination__item\\">1</div><div class=\\"app__blog__list__pagination__item\\">2</div><div class=\\"app__blog__list__pagination__item\\">3</div><div class=\\"app__blog__list__pagination__item\\">4</div><div class=\\"app__blog__list__pagination__item\\">5</div><div class=\\"app__blog__list__pagination__item abbr\\">...</div><div class=\\"app__blog__list__pagination__item\\">12</div><div class=\\"app__blog__list__pagination__item next\\">&gt;</div></div></div></section></div></div>"`;
